react-components/src/components/card/Card.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './cards.scss';
+import { CardItemModel } from '../Models/CardItemModel';
+
+type CardProps = CardItemModel;
+
+const Card: React.FC<CardProps> = (props: CardProps) => {
+  return (
+    <div className={'card'}>
```

Comment:
If you use string className you can use only this string:
`<div className={'card'}>` => `<div className="card">`

Fix everywhere